### DISCUSSION TITLE
Way to enable tiling mode from terminal?

### DISCUSSION THREAD

**Discussion by halfmexican** [pop-os/shell#1591] [Q&A]
So I have a really specific issue, I want to know if there is a way to enable tilling from the terminal. I was looking at using DBUS commands but I can't figure it out. I want bind the tilling mode to a touchegg gesture

**Comment by jacobgkau** [comment]
I think the gsetting in question might be `tile-by-default`-- it seems to be set to `true` when tiling is enabled and `false` when tiling is disabled. However, setting it to `true`/`false` using `gsettings set org.gnome.shell.extensions.pop-shell tile-by-default ...` doesn't seem to work properly.

> I want bind the tilling mode to a touchegg gesture

Can you just bind a gesture to the Super + Y keyboard shortcut for toggling tiling?
